ci/build_test_OnCommit.groovy

```diff
@@ -82,6 +82,7 @@ pipeline {
               when { expression { image_build_needed == "true" } }
                 steps {
                       sh "echo build --remote_cache=${env.OVMS_BAZEL_REMOTE_CACHE_URL} > .user.bazelrc"
+                      sh "echo test:linux --test_env https_proxy=${env.HTTPS_PROXY} > .user.bazelrc"
```

Comment:
Wouldn't this overwrite effect of the command above? ">>" probably should be used